downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -12,14 +12,26 @@
 
     private static final Logger logger = LogManager.getLogger();
 
+    // CrossReference IDs of species containing an alternative GO compartment, which do not receive a GO annotation: HIV 1, unknown, C. botulinum, B. anthracis
```

Comment:
Could the species identifiers be named constants (maybe in their own file)?  That way each numeric id is associated with its specific species name and the list in the code won't get out of sync with the list in the comment.

Reply:
Done